components/event/EventCard.js

```diff
@@ -35,6 +35,28 @@ export default function EventCard({ event, username }) {
             )}
         </div>
         <div className="flex-1 space-y-1">
+          {!username && event.userStatus && (
+            <div className="flex space-x-2">
+              <p>
+                {name} is{" "}
+                <span
+                  className="font-bold"
+                  style={{
+                    color: event.color,
+                  }}
+                >
+                  {event.userStatus}
+                </span>{" "}
+                {event.userStatus == "speaking" && "at"} this event.
+              </p>
+              {event.userStatus == "speaking" && (
+                <p>
```

Comment:
There is a need of condition here as well as `speakingTopicDetails` is also needed

`{event.userStatus == "speaking" && event.speakingTopicDetails && (
                 <p>
                   <span className="font-bold">Topic: </span>{" "}
                   {event.speakingTopicDetails.topic} 
                 </p>
              )}`

Reply:
Topic details are not necessary I think.

Reply:
yeah, but according to the code if the `SpeakingTopicDetails` is not present then it just shows `Topic :`

Reply:
Ohk I got it